srcpkgs/AppStream/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'AppStream'
 pkgname=AppStream
-version=0.12.11
-revision=2
+version=0.13.1
+revision=1
 wrksrc="appstream-${version}"
 build_style=meson
 build_helper="gir qemu"
```

Comment:
Why is the `qemu` build helper required? I see nothing running in under emulation in this template.

Reply:
the `meson` build style can use qemu automatically. But I think specifying `gir` should be enough.

Reply:
> Why is the `qemu` build helper required? I see nothing running in under emulation in this template.

Without it the cross builds were failing with:
```
Cannot use target appstreamcli as a generator because it is built for the
host machine and no exe wrapper is defined or needs_exe_wrapper is
true. You might want to set `native: true` instead to build it for
the build machine.
```

Reply:
No problem, thanks for checking.